.github/workflows/lint-spellcheck.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint Spell Check
```

Comment:
This can be part of `lint-global.yml`, right?

Reply:
I don't know. it has to run on all files and not a subsection like dprint.

Reply:
It should be resolved now.